cms/lib/xblock/tagging/tagging.py

```diff
@@ -71,25 +79,36 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
         """
         Handler to save choosen tags with connected XBlock
         """
-        found = False
-        if 'tag' not in request.params:
-            return Response("The required parameter 'tag' is not passed", status=400)
+        try:
+            posted_data = json.loads(request.body_file.read())
```

Comment:
Nit: you should be able to use `request.json` to get this: http://docs.webob.org/en/stable/api/request.html#webob.request.BaseRequest.json